src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,53 @@
+import { Link, NavLink } from "react-router-dom";
+import "./sidebar.css";
+
+const sidebarData = [
```

Comment:
This is really nice that you have created an array and then mapped over it. And not only this, you have also declared it outside the component so that it does not get created on each render that's great.

Reply:
Thanks